packages/core/echo/echo/src/type/index.ts

```diff
@@ -2,5 +2,7 @@
 // Copyright 2025 DXOS.org
 //
 
-export { Type } from './Type';
+export * as Obj from './Obj';
```

Comment:
Those seem like a nice set of top-level nouns

Is the intent that Obj is the common base for Type and Relation?

I still don't like Type in here as it's hard to call instances of those objects "types". How about renaming it to Entity?

Reply:
I think we need to decide if we can live with `Obj.Obj.Any` which is an outcome of the effect pattern.

Reply:
I think the way to clean that up is not importing from the index. In cases when you're just working w/ types you can import from `@dxos/echo/Object`